test/integration_test/basic_test.go

```diff
@@ -352,6 +380,12 @@ func BasicUpgradeStorageCluster(tc *types.TestCase) func(*testing.T) {
 			lastHopURL = hopURL
 		}
 
+		// Delete AlertManager secret, if found
```

Comment:
can we do this inside of `UninstallAndValidateStorageCluster`? check if alert manager is enabled then delete the secret. as I noticed the migration test may leave stale alertmanager secret as well.

Reply:
Done